### PR TITLE
Build managed code and packages for WebAssembly/netcoreappaot

### PR DESCRIPTION
**Pull request by morganbr**
Adds support for a WebAssembly netcoreappaot configuration for building managed libraries and packages (native already builds). This is enough to produce a NuGet package of libraries that CoreRT's WebAssembly implementation can consume.

### CODE REVIEW

**Comment by morganbr** [comment]
The build issue is just due to a change that will be in the next ProjectN build on Sunday night. This is ready to review (and we just won't merge it until we get the new build).

File: src/System.Xml.XmlSerializer/src/Configurations.props [hunk 1]
```diff
@@ -5,7 +5,9 @@
       uap-Windows_NT;
       uapaot-Windows_NT;
       netcoreapp-Windows_NT;
+      netcoreappaot-Windows_NT;
```
Comment (ericstj) [inline src/System.Xml.XmlSerializer/src/Configurations.props]: Why adding netcoreappaot-Windows_NT here?

**Comment by ericstj** [comment]
This looks ok.  One thing that we could do to avoid having to change so many projects is to have some sort of shortcut for the set of configurations that mirror the set of unique runtimes.  Then we could have the partial-facade projects use that property rather than duplicating the list everywhere.

**Comment by morganbr** [comment]
@dotnet-bot test Packaging All Configurations x64 Debug Build

**Comment by ericstj** [comment]
@morganbr I haven't seen a maestro update for ProjectN bits in a [while](https://github.com/dotnet/corefx/pull/31188).  Are these flowing?

**Comment by morganbr** [comment]
Yep, they're flowing, but the source change didn't get mirrored from CoreRT to TFS yet. I'll just put in an APICompatBaseline file.

**Comment by weshaggard** [comment]
@morganbr what is the current plans for webassembly? I mostly ask as I want us to avoid shipping webassembly specific assets in our nuget packages if we don't have full support for them yet. We do the same for UWP assets were we disable them before we release.

**Comment by morganbr** [comment]
Based on our offline discussion, I'll go through and confirm whether any WebAssembly or netcoreappaot files end up in the packages for individual libraries (and remove them if they do).

**Comment by morganbr** [comment]
I've confirmed the packages look good. Thanks for the reviews!

**Pull request merged**